pkg/syncer/controller/syncerconfigmanager.go

```diff
@@ -129,6 +132,99 @@ func (s *SyncerConfigManager) upsertNamespaceScoped(syncerConfig edgev1alpha1.Sy
 	s.syncConfigManager.upsert(edgeSyncConfig)
 }
 
+type typeFlatNamespacedObject struct {
+	APIVersion string
```

Comment:
Since "APIVersion" is commonly used elsewhere in the Kubernetes milieu to mean the combination of API group and version, it is worth a comment here saying that only the version part appears in this field.